wdl/tasks/stampipes.wdl

```diff
@@ -168,3 +168,35 @@ task cutfragments {
         disks: resources.disks
     }
 }
+
+
+task chrom_buckets {
+    input {
+        File fai
+        Int bin_size = 20
+        Int window_size = 75
```

Comment:
Should only bind these at the subworkflow level.

Reply:
Good point.